### PR TITLE
feat: configure ccd shutter

### PR DESCRIPTION
**Pull request by lewisbirks**
### JIRA link (if applicable) ###

[FPLA-1892](https://tools.hmcts.net/jira/browse/FPLA-1892)

### Change description ###

Added the shutter functionality to our jurisdiction based off the [confluence guide](https://tools.hmcts.net/confluence/display/RCCD/CCD+UI+Shuttering)
Updated charts for preview.
[Change in FPLA-Docker](https://github.com/hmcts/fpla-docker/pull/17)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/Jurisdiction.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "PUBLICLAW",
     "Name": "Public Law",
-    "Description": "Public Law"
+    "Description": "Public Law",
+    "Shuttered": "Yes"
```
Comment (KiranVarma02) [inline ccd-definition/Jurisdiction.json:7]: Just FMI, Does it shutter service when we release this?

Comment (lewisbirks) [reply ccd-definition/Jurisdiction.json:7]: yeah I think so

**Comment by lewisbirks** [comment]
Should I reset the fpla-docker submodule here and then wait for it to be merged in that repo and the changes to then be merged for this one?

**Comment by KiranVarma02** [comment]
> Should I reset the fpla-docker submodule here and then wait for it to be merged in that repo and the changes to then be merged for this one?

I commented on fpla-docker PR, Just want to understand why we would want to shutter local?

File: ccd-definition/Jurisdiction.json (hunk 1 above)
Comment (akomisarek) [reply ccd-definition/Jurisdiction.json:7]: Should we put this nonprod for the time being?

Comment (lewisbirks) [reply ccd-definition/Jurisdiction.json:7]: Can do

Comment (lewisbirks) [reply ccd-definition/Jurisdiction.json:7]: Will just need a small ticket to toggle on friday

Comment (lewisbirks) [reply ccd-definition/Jurisdiction.json:7]: 🤔 seems that it isn't as simple as just duplicating and adding `-nonprod` to the file name as both are still read locally and there can only be one jurisdiction

**Pull request merged**